.sharing.io/values.template.yaml

```diff
@@ -0,0 +1,42 @@
+coder:
+  # You can specify any environment variables you'd like to pass to Coder
+  # here. Coder consumes environment variables listed in
+  # `coder server --help`, and these environment variables are also passed
+  # to the workspace provisioner (so you can consume them in your Terraform
+  # templates for auth keys etc.).
+  #
+  # Please keep in mind that you should not set `CODER_ADDRESS`,
+  # `CODER_TLS_ENABLE`, `CODER_TLS_CERT_FILE` or `CODER_TLS_KEY_FILE` as
+  # they are already set by the Helm chart and will cause conflicts.
+  image:
+    repo: "cdrdean/coder"
+    tag: "v0.13.1-devel-343ce356-amd64"
```

Comment:
This is part of the fix for emacs-broadway https://github.com/coder/coder/pull/5334